extensions/amp-ad-network-doubleclick-impl/0.1/safeframe-host.js

```diff
@@ -160,6 +160,12 @@ export class SafeframeHostApi {
     /** @private {?({width, height}|../../../src/layout-rect.LayoutRectDef)} */
     this.creativeSize_ = creativeSize;
 
+    /** @private {?({width, height}|../../../src/layout-rect.LayoutRectDef)} */
+    this.initialCreativeSize_ = {
+      height: creativeSize['height'],
+      width: creativeSize['width'],
+    };
+
```

Comment:
I don't think it matters much, but I'd expect `getInitialGeometry` to use `initialCreativeSize_` and not `creativeSize_`

Reply:
good point, changed